pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerImpl.java

```diff
@@ -1292,6 +1293,12 @@ protected OpSendMsg newObject(Handle<OpSendMsg> handle) {
         };
     }
 
+    private static double nsToSeconds(long ns) {
```

Comment:
I would expect this find this method in a utility class

Reply:
Thanks @nicoloboschi for your advice. I agree with you, but so far there is no suitable utility class such as `TimeUtils`. Maybe later, when there are more methods like `#nsToSeconds`, we can create one.

Reply:
There's always a starting point and perhaps there were other times when this argument has been exposed and now we haven't created the `TimeUtils` class yet

More in general, duplicating code is a serious problem and can be lead to a security problems. A utility class with single method is always better to a duplicate method

Reply:
Thanks for your reply, I will create a `TimeUtils` ASAP.

Reply:
I found `RelativeTimeUtil` and moved `#nsToSeconds` to it. @nicoloboschi PTAL. Thanks a lot.